File: Task2/answer.c
#include <stdio.h>

int main() {
    for (int i = 1; i <= 100; i++) {
        if (i % 3 == 0 && i % 5 == 0 && i % 7 == 0) {
            printf("Hello World Yoo\n");
        } else if (i % 3 == 0 && i % 5 == 0) {
            printf("Hello World\n");
        } else if (i % 3 == 0 && i % 7 == 0) {
            printf("Hello Yoo\n");
        } else if (i % 5 == 0 && i % 7 == 0) {
            printf("World Yoo\n");
        } else if (i % 3 == 0) {
            printf("Hello\n");
        } else if (i % 5 == 0) {
            printf("World\n");
        } else if (i % 7 == 0) {
            printf("Yoo\n");
        } else {
            printf("%d\n", i);
        }
    }
    return 0;
}

File: Task2/answer.js
for (let i = 1; i <= 100; i++) {
  if (i % 3 === 0 && i % 5 === 0 && i % 7 === 0) {
    process.stdout.write("Hello World Yoo");
  } else if (i % 3 === 0 && i % 5 === 0) {
    process.stdout.write("Hello World");
  } else if (i % 3 === 0 && i % 7 === 0) {
    process.stdout.write("Hello Yoo");
  } else if (i % 5 === 0 && i % 7 === 0) {
    process.stdout.write("World Yoo");
  } else if (i % 3 === 0) {
    process.stdout.write("Hello");
  } else if (i % 5 === 0) {
    process.stdout.write("World");
  } else if (i % 7 === 0) {
    process.stdout.write("Yoo");
  } else {
    process.stdout.write(i.toString());
  }
  if (i < 100) {
    process.stdout.write(", ");
  }
}
